hypothesis-python/docs/reference/index.rst

```diff
@@ -0,0 +1,11 @@
+API Reference
+=============
+
+The technical API reference for Hypothesis is split into three pages. The strategies reference documents builtin strategies, including for :doc:`extras </extras>`. The API reference documents non-strategy objects that are part of the public API. The integrations reference documents Hypothesis features with a defined interface, but no code API.
+
+.. toctree::
+   :maxdepth: 1
+
+   strategies
+   api
```

Comment:
reverse these?

Reply:
I'd guess that most people who look up the hypothesis docs are looking up a strategy! I guess the ordering is a bit awkward with the sidebar heading being "API Reference" and then its nested "API Reference" page being second, not first.